Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp

```diff
@@ -1597,6 +1629,12 @@ namespace AzToolsFramework
                 GUI->SetEditButtonTooltip(tr(buttonTooltip.c_str()));
             }
         }
+        else if (attrib == AZ_CRC_CE("DisableEditButtonWheNoAssetSelected"))
+        {
+            bool disableEditButtonWheNoAssetSelected = false;
+            attrValue->Read<bool>(disableEditButtonWheNoAssetSelected);
+            GUI->SetDisableEditButtonWheNoAssetSelected(disableEditButtonWheNoAssetSelected);
+        }
```

Comment:
question: This looks very similar to line `1378`, could be moved to a helper, not a huge deal though